insights/specs/default.py

```diff
@@ -311,6 +311,27 @@ def httpd_cmd(broker):
         # https://access.redhat.com/solutions/21680
         return list(ps_httpds)
 
+    @datasource(Mount)
+    def httpd_on_nfs(broker):
+        mnt = broker[Mount]
+        mps = mnt.search(mount_type='nfs4')
+        # get nfs 4.0 mount points
+        nfs_mounts = [m.mount_point for m in mps if m['mount_options'].get("vers") == "4.0"]
+        if nfs_mounts:
+            # get all httpd ps
+            out = broker[HostContext].shell_out("pgrep httpd")
+            httpd_pids = out.splits("\n")
```

Comment:
This `splits` is not necessary, since out is already `splitlines`-ed:


https://github.com/RedHatInsights/insights-core/blob/a442128998c39413f674a995dd81d0e493805704/insights/core/context.py#L149


Related traceback:
~~~
  File "/root/insights_dev/insights-core/insights/specs/default.py", line 324, in httpd_on_nfs
    httpd_pids = out.splits("\n")
AttributeError: 'list' object has no attribute 'splits'
~~~